tasks/main.yml

```diff
@@ -25,7 +25,7 @@
   meta: flush_handlers
 
 - include_tasks: docker-compose.yml
-  when: docker_install_compose
+  when: docker_install_compose | bool
 
 - include_tasks: docker-users.yml
-  when: docker_users
+  when: docker_users | bool
```

Comment:
The only thing here is I'd like to have this be a test for an empty array instead of a cast to bool. I can't think of it off the top of my head but there is probably a corner case where a valid array could evaluate to false when casting it...